cs/keycloak/variables.tf

```diff
@@ -51,3 +51,12 @@ variable "keycloak_secrets_arn" {
   description = "ARN of the Keycloak secrets manager"
   sensitive   = false
 }
+
+variable "keycloak_task_size" {
+  type = object({
+    cpu    = any
```

Comment:
shouldn't these be `number`s? (same for the higher levels of variable declarations)